test_runner/fixtures/compare_fixtures.py

```diff
@@ -333,3 +338,27 @@ def neon_with_baseline(request: FixtureRequest) -> PgCompare:
     fixture = request.getfixturevalue(request.param)
     assert isinstance(fixture, PgCompare), f"test error: fixture {fixture} is not PgCompare"
     return fixture
+
+
+@pytest.fixture(scope="function", autouse=True)
+def sync_after_each_test():
+    # some of the benchmarks are quite write-happy, and create issues to start the processes up in 10s.
+    # SYNC_AFTER_EACH_TEST is only "true" for benchmarks, which run test_runner/performance cases sequentially
+    key = "SYNC_AFTER_EACH_TEST"
+    enabled = os.environ.get(key) == "true"
+    yield
+    if not enabled:
+        log.info("not calling sync after test")
+        return
+
+    # sync is part of coreutils, and we only run benches on linuxes
+    path = Path("/usr/bin/sync")
```

Comment:
Why not use [`os.sync()`](https://docs.python.org/3/library/os.html#os.sync)?
If we decide to call a separate binary, I'd use [`shutil.which`](https://docs.python.org/3/library/shutil.html#shutil.which) instead of hardcoded path

Reply:
good idea

Reply:
Ok, it does what it should do: https://github.com/python/cpython/blob/56b00f4705634af2861a8aa9c2eb5769012220f0/Modules/posixmodule.c#L3868-L3876

I wish python had some docs, or at least cpython would be more accessible.